averbis/core/_rest_client.py

```diff
@@ -1803,6 +1827,17 @@ def _delete_pipeline(self, project: str, pipeline: str) -> dict:
         )
         return response["payload"]
 
+    def _list_pipelines(self, project: str) -> dict:
+        """
+        HIGHLY EXPERIMENTAL API - may soon change or disappear.
+
+        Use Project.list_pipelines() instead.
+        """
+        response = self.__request_with_json_response(
```

Comment:
Should we add a check here to raise an Exception if it is a platform version 5.x?